api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionUserInstructions.java

```diff
@@ -67,7 +67,7 @@ public boolean equals(Object o) {
 
     DbInstitutionUserInstructions that = (DbInstitutionUserInstructions) o;
 
-    return Objects.equals(institution.getInstitutionId(), that.institution.getInstitutionId())
+    return Objects.equals(institution, that.institution)
```

Comment:
This is now an equivalence comparison (recursing through the Institution model fields) rather than "is this exactly the same row in the DB"